NinjaResidence/DirectX.cpp

```diff
@@ -245,6 +244,10 @@ void DirectX::CheckKeyStatus() {
 	}
 }
 
+bool DirectX::PushAnykey() {
+	BYTE* tmp = std::find_if(std::begin(m_KeyState), std::end(m_KeyState), [](BYTE m_KeyState) {return m_KeyState; });
```

Comment:
* ラムダ式の仮引数は `DirectX::m_KeyState` では無くその中の各要素のはずなので名前変えましょう
* ラムダ式の戻り値は、何をチェックして真偽を判断しているのかを明示した方がわかりやすいと思います
```suggestion
	BYTE* tmp = std::find_if(std::begin(m_KeyState), std::end(m_KeyState), [](BYTE keyState) { return keyState != 0; });
```
こんな感じの方が良いと思います。